Sources/NIOPosix/ControlMessage.swift

```diff
@@ -226,6 +228,8 @@ struct ControlMessageParser {
     }
 
     private mutating func receiveIPv4Message(_ controlMessage: UnsafeControlMessage) {
+#if os(FreeBSD)
+#else
```

Comment:
Is anything going to go in this first block? If not, invert the condition and remove the `else`.

Reply:
no idea why GitHub is still showing nothing inside... it should show setting the control messages here